wrappers/services_gen_test.go

```diff
@@ -202,15 +202,15 @@ apps:
 	defer restore()
```

Comment:
we need new tests here about ServiceQuotaMap being set, no?

Reply:
I might be missing something here, but we have tests for ServiceQuotaMap behaviour in services_test.go, and it seems the tests here in services_gen_test.go are very focused on `GenerateSnapServiceFile` which doesn't use ServiceQuotaMap, so I'm not sure it's relevant?